app/controllers/admin/users_controller.rb

```diff
@@ -8,6 +8,12 @@ def index
     @users = @search.result
   end
   
+  def show
+    @total_questions = Question.count
+    @total_topics = Topic.count
+    @users = User.all    
```

Comment:
also i don't really understand which route you rely on

currently we have  `resources :users (in admin namespace)` and it generates `admin/users/:id` which applies to current action and require some :id